ipa-core/src/protocol/ipa_prf/mod.rs

```diff
@@ -178,7 +192,7 @@ pub mod tests {
                     trigger_value: 0,
                 },
                 TestRawDataRecord {
-                    timestamp: 0,
+                    timestamp: 5,
```

Comment:
I don't think we should change the timestamps.

Reply:
This test says 2 sources have same timestamp and it should be attributed to 2nd breakdown which sounds a bit absurd

Reply:
Fair enough! Invalid input.